grafonnet/stackdriver.libsonnet

```diff
@@ -0,0 +1,50 @@
+{
+  /**
+   * Creates a [Stackdriver target](https://grafana.com/docs/grafana/latest/datasources/cloudmonitoring/)
+   *
+   * @name stackdriver.target
+   *
+   * @param projectName
+   * @param aliasBy
+   * @param metricKind
+   * @param metricType
+   * @param alignmentPeriod (default: `'stackdriver-auto'`)
+   * @param crossSeriesReducer
+   * @param perSeriesAligner
+   * @param groupBys (default: `[]`)
+   * @param filters (default: `[]`)
+   * @param unit
+   * @param valueType
+
+   * @return Panel target
+   */
+
+  target(
```

Comment:
fyi: specs of plugin also mention a `view` param .. though no idea if that is obsolete, or a new feature https://github.com/grafana/grafana/blob/aae4ca754497ebfdbf70560596cd9185fd787b21/public/app/plugins/datasource/cloud-monitoring/types.ts#L67